packages/react/src/components/molecules/BrandBanner/index.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * BrandBanner module.
+ * @module @massds/mayflower-react/BrandBanner
+ * @requires module:@massds/mayflower-assets/scss/02-molecules/brand-banner
+ * @requires module:@massds/mayflower-assets/scss/01-atoms/button-with-icon
+ */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Image from 'MayflowerReactMedia/Image';
+import ButtonWithIcon from 'MayflowerReactButtons/ButtonWithIcon';
+import IconChevron from 'MayflowerReactBase/Icon/IconChevron';
+import IconBuilding from 'MayflowerReactBase/Icon/IconBuilding';
+import IconLock from 'MayflowerReactBase/Icon/IconLock';
+import logo from '@massds/mayflower-assets/static/images/logo/stateseal.png';
```

Comment:
You shouldn't import files here. Instead, require the user to pass in the src of the image they want to use.

Reply:
ah ok.. so there's no way to enforce passing in the image of the seal then right?